app/src/main/java/com/imdbmovieapp/data/remote/dto/detailMovieDto/MovieDetailDto.kt

```diff
@@ -0,0 +1,13 @@
+package com.imdbmovieapp.data.remote.dto.detailMovieDto
+
+import com.imdbmovieapp.data.remote.dto.genreMovieDto.Genre
+
+data class MovieDetailDto(
+    val genres: List<Genre>,
+    val id: Int,
+    val overview: String,
+    val poster_path: String,
+    val release_date: String,
+    val title: String,
+    val vote_average: Double,
```

Comment:
Try not to use commas for the last parameters. You have it in MoviesResult as well. It's a matter of style, but when you will be in Space project, you will have to fix bot comments about it :D